src/components/RescueDetails.tsx

```diff
@@ -275,6 +307,14 @@ export default function RescueDetails({id}: { id: string }) {
         }
     }
 
+    function renderSecondVolunteerElements(rescue : BirdAlert) {
+        if (rescue.secondVolunteer && rescue.twoPersonRescue) {
+            return `, ${rescue.secondVolunteer.replace(",", "").split(' ').reverse().join(' ')}`
+        }else if (!rescue.secondVolunteer && rescue.twoPersonRescue && rescue.currentVolunteer) {
+            return `, SECOND VOLUNTEER NEEDED`
```

Comment:
I wonder if @V-Downs would want to give a little UX direction on this, that we could address in a follow up

Reply:
I would definitley like to get some feedback on the flow for the name section, I did it this way since when there are two names it would be "firstLastName, firstFirstName, secondLastName, firstLastName" which is a bit confusing to read. But there may be a better way to do it.